microovn/frr/bgp/persistance.go

```diff
@@ -35,8 +35,9 @@ func EnsureInterfacesInVrf(ctx context.Context, s state.State) error {
 		}
 		portVrf = strings.TrimSpace(portVrf)
 
-		portIP, err := ovnCmd.VSCtl(ctx, s, "get", "port", port, fmt.Sprintf("external-ids:%s", BgpIfaceIP))
-		if err != nil {
+		portIP := ""
+		portIP, err = ovnCmd.VSCtl(ctx, s, "get", "port", port, fmt.Sprintf("external-ids:%s", BgpIfaceIP))
+		if err == nil {
```

Comment:
Is this condition correct? Wouldn't that mean that when `VSCtl` successfully finds interface's IP in the `external-ids`, we'd log an error?